src/main/java/uk/gov/hmcts/darts/arm/model/rpo/AbstractMatterResponse.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.darts.arm.model.rpo;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@NoArgsConstructor
+public abstract class AbstractMatterResponse {
```

Comment:
Do all ARM calls have the same response?

Reply:
Most of the RPO responses will return this object which is why this was abstracted out.